docker/Dockerfile

```diff
@@ -7,7 +7,7 @@ MAINTAINER andreas-bok-sociomantic <andreas.bok@sociomantic.com>
 # Set environment variables
 ENV MYSQL_USER=user \
     MYSQL_PASSWORD=pass \
-    MYSQL_ROOT_PASSWORD='' \
+    MYSQL_ROOT_PASSWORD=pass \
```

Comment:
Why "pass"? If this is supposed to be a dummy password, wouldn't it be better to make it more self-documenting as `MYSQL_ROOT_PASSWORD=dummy_password` or something like that?